src/Header.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+
+function Header() {
+  return (
+    <Wrapper>
+      <Image src="/img/Schriftzug.jpg" alt="" />
```

Comment:
Schriftzug is a very generic, maybe simply Logo? Think about adding an alt text as this is the header of your app and you I think you want something to be displayed in case the image fails to load.

There is an easy and robust way to import static images https://create-react-app.dev/docs/adding-images-fonts-and-files/